packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1134,22 +1103,22 @@ contract VotingV2 is
         bytes memory ancillaryData
     ) public {
         EmergencyAction storage action = emergencyActions[_encodePriceRequest(identifier, time, ancillaryData)];
-        require(action.accountSignaled[msg.sender] == 0, "Already signaled");
+        require(action.accountSignaled[msg.sender] == 0);
 
         action.accountSignaled[msg.sender] = getVoterStake(msg.sender);
         action.cumulativeSignaled += action.accountSignaled[msg.sender];
 
         // ++ the num of emergency actions this staker has signaled to block unstaking unless all are canceled.
         stakerEmergencyActionSignalCount[msg.sender]++;
 
-        // Block the staker from callin requestUnstake or executeUnstake by setting their unstake trackers to values
+        // Block the staker from calling requestUnstake or executeUnstake by setting their unstake trackers to values
         // that will mature far in the future. Remember that this method is meant to be used only in an emergency
         // and we want to block callers from doing things like: 1) staking, 2) signaling 3) unstaking and
         // 4) re-staking from another wallet to multiply their voting power. If you signal you must be blocked. This is
         // only done on the first signal of emergency action to not unnecessarily increment these trackers.
         if (stakerEmergencyActionSignalCount[msg.sender] == 1) {
             voterStakes[msg.sender].pendingUnstake += 1; // prevents staker from calling requestUnstake.
-            voterStakes[msg.sender].unstakeRequestTime += 4813802983; // 100 years from now.
+            voterStakes[msg.sender].unstakeRequestTime += 10000000000; // A long time in the future.
```

Comment:
nit: can we make this a constant, since we use it in two places and it's vital that it's identical?